src/channel/listener.rs

```diff
@@ -402,6 +400,23 @@ impl Listener {
         ShutdownError::SocketShutdown
     }
 
+    fn push_received(&self, joined: Joined, push: Push) -> State {
+        debug!(
```

Comment:
Did you want `debug` here or something like `tokio::trace`?

Reply:
The whole library uses debug! for the logger.  If you think tokio::trace would be better I'd prefer to switch over in a different PR as a whole.  What are the benefits of tokio::trace over the current debug?

Reply:
I actually mistook `debug` for the [`dbg`](https://doc.rust-lang.org/std/macro.dbg.html).